package.json

```diff
@@ -84,7 +85,8 @@
     "react-image": "^1.3.1",
     "react-imageloader": "2.1.0",
     "react-measure": "^1.4.7",
-    "react-monaco-editor": "^0.7.3",
+    "monaco-editor": "^0.9.0",
```

Comment:
I'm not sure why this dependency is in the package json.  I need to track where it has come from.

We should not be pulling in the monaco-editor, I'm sure the react-monaco-editor package should....

Reply:
will debug

Reply:
For some reason with react-monaco-editor it does not install the monaco-editor dependency for itself. 
The npm installation logs gave me a message of monaco-editor@0.9.0 expected

Reply:
it's cool it brings in the monaco editor as a peer dep - in the 0.9.x range, so we good for now.